release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2780,3 +2780,30 @@ link:https://access.redhat.com/solutions/6934361[{product-title} 4.9.29 containe
 ==== Updating
 
 To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-31"]
+=== RHBA-2022:1605 - {product-title} 4.9.31 bug fix update
+
+Issued: 2022-05-03
+
+{product-title} release 4.9.31, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:1605[RHBA-2022:1605] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:1604[RHBA-2022:1604] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6955866[{product-title} 4.9.31 container image list]
+
+[id="ocp-kubernetes-1-22-8-updates"]
+==== Updates from Kubernetes 1.22.8
+
+This update contains changes from Kubernetes 1.22.6 up to 1.22.8. More information can be found in the following changelogs: link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#v1226[1.22.6], link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#v1227[1.22.7], and link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#v1228[1.22.8].
+
+[id="ocp-4-9-31-bug-fixes"]
+==== Bug fixes
+* Previously, no warning was provided in {product-title} 4.9 about API requiring users to specify either `ipFamilyPolicy: PreferDualStack` or `ipFamilyPolicy: RequireDualStack` for dual-stack services to be valid in {product-title} 4.10. This fix alerts users of the upcoming API change, ensuring awareness of the required specifications. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2050632#[*BZ#2050632*])
```

Comment:
This is a little bit hard for me to track. And this seems like tough bug to describe. In particular, the use of passive voice is kind of throwing me. WDYT of something like this:
```suggestion
* Previously, the Kubernetes API server did not warn users about upcoming changes to the API for dual-stack services in {product-title} 4.10. This fix alerts users to specify either `ipFamilyPolicy: PreferDualStack` or `ipFamilyPolicy: RequireDualStack` for dual-stack services to be valid in {product-title} 4.10. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2050632#[*BZ#2050632*])
```

Reply:
Yes, this works well! I intended to make it as easily understood as possible. Thank you so much! I have made the change :)